packages/manager/modules/vrack/src/terminate-modal/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "vrack_terminate": "Supprimer",
+  "vrack_terminate_title": "Supprimer mon Vrack",
+  "vrack_terminate_description": "Afin de supprimer votre Vrack vous devez au préalable deconnecter tous les services qui y sont ratachés",
```

Comment:
You miss the implementation for this line

Reply:
Its available in vrack.html page I need to push the implementation on disable button once 2api changes are done.